Content.Shared/Hands/Components/SharedHandsComponent.cs

```diff
@@ -11,6 +11,7 @@ public abstract class SharedHandsComponent : Component
     ///     The currently active hand.
     /// </summary>
     [ViewVariables]
+    [DataField("pssActiveHand")]
     public Hand? ActiveHand;
```

Comment:
Instead of storing a copy of the hand data, there should just be a data-field with the hand id/name along with code that sets ActiveHand on startup.